src/System.Private.ServiceModel/src/System/ServiceModel/Dispatcher/DataContractSerializerOperationFormatter.cs

```diff
@@ -35,10 +35,11 @@ internal static DataContractSerializer CreateSerializer(Type type, string rootNa
 
         internal static DataContractSerializer CreateSerializer(Type type, IList<Type> knownTypes, string rootName, string rootNs, int maxItems)
         {
+            XmlDictionary dictionary = new XmlDictionary(2);
             return new DataContractSerializer(
                 type,
-                rootName,
-                rootNs,
+                dictionary.Add(rootName),
+                dictionary.Add(rootNs),
```

Comment:
I already opened the issue. Because of the delay in the issue getting fixed and the fix being available from nuget, waiting for the fix will leave our scenario broken for a while. Customers would get a null reference exception and our tests would fail.

Sent from my Windows Phone
________________________________
From: Zhenlan Wang<mailto:notifications@github.com>
Sent: ‎7/‎23/‎2015 10:18 PM
To: dotnet/wcf<mailto:wcf@noreply.github.com>
Cc: Matt Connew<mailto:mconnew@microsoft.com>
Subject: Re: [wcf] Fixing callback exception handling and adding tests (#216)


In src/System.Private.ServiceModel/src/System/ServiceModel/Dispatcher/DataContractSerializerOperationFormatter.cs<https://github.com/dotnet/wcf/pull/216#discussion_r35396116>:

>              return new DataContractSerializer(
>                  type,
> -                rootName,
> -                rootNs,
> +                dictionary.Add(rootName),
> +                dictionary.Add(rootNs),


@mconnew<https://github.com/mconnew> can you please open an issue in corefx against DCS? No need to change code here.

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/wcf/pull/216/files#r35396116>.